accounts-db/src/in_mem_accounts_index.rs

```diff
@@ -672,14 +672,16 @@ impl<T: IndexValue, U: DiskIndexValue + From<T> + Into<T>> InMemAccountsIndex<T,
 
     /// Queue up these insertions for when the flush thread is dealing with this bin.
     /// This is very fast and requires no lookups or disk access.
-    pub fn startup_insert_only(&self, slot: Slot, items: impl Iterator<Item = (Pubkey, T)>) {
+    pub fn startup_insert_only(&self, items: impl Iterator<Item = (Pubkey, (Slot, T))>) {
```

Comment:
yes and no. First, see my comment below. we already add the slot shortly after this. We have 432k slots, 8192 bins, 340M accounts. The index entries are added async later. They need to be sorted by bin, then by hash(pubkey + random) within the bin. This mixes slots all around. And the slot is paired with `T`/`U`/`v` or whatever in the index. So, yes, we will be adding the slot later. But the idea here is to memcpy the pubkey + (slot, T) into the `insert` vec for later async processing instead of arbitrarily reformatting it.